markdown/solace-pubsub-microgateway/solace-pubsub-microgateway.md

```diff
@@ -0,0 +1,1185 @@
+author: Aaron Lee
+summary: Summary of your codelab that is human readable
+id: solace-pubsub-microgateway
+tags: iguide
```

Comment:
Okay so we have a decision to make here. This tag is used for filtering. Right now we are able to filter on workshop for Developer Workshops and iguide for Integration Guides, but I think we need a third option for our "How To Recipes for Design Patterns" so I think we need to come up with a name? Do we just call them "Design Patterns" or "How To Recipes" or something else? I'll think about this some too...

Reply:
"howto" ..?   or "patterns" ..?  This one kinda falls into both.,  But there will be some that is one and not the other.  I'm assuming we can tag it with both?  E.g.: "howto,patterns"

Reply:
Yes, that's fine. Let's put both.

Reply:
As discussed, adding two new categories: "patterns == Design Patterns", and "howto == How To Guides"  (or something like that).